Create personal page

austinbrehm opened the pull request:
## Description
Created a personal page: 

<img width="1490" alt="personal_page" src="https://github.com/user-attachments/assets/804161f0-d3fe-41ee-abed-5193e0648623" />

## Additional Information
- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues
_Closes #8_

Your ENS/address: brehm.eth

phipsae commented on packages/nextjs/app/builders/brehm.eth/page.tsx:
```diff
@@ -0,0 +1,80 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const Profile: NextPage = () => {
```
Since there are more builder profiles, it might make sense to give your component a more specific name (e.g. "YourNameProfile")

phipsae commented on packages/nextjs/app/builders/brehm.eth/page.tsx:
```diff
@@ -0,0 +1,80 @@
+          <div className="text-center"></div>
+        </div>
+
+        <div className="mt-6">
+          <div className="space-y-2">
+            <Link
+              href="https://app.buidlguidl.com/builders/0xeD873463fb62E2B47BE045beA7385d35166A56dE"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="block p-2 hover:bg-base-300 rounded-lg"
+            >
+              Buidl Guidl
+            </Link>
+          </div>
+
+          <div className="space-y-2">
+            <Link
+              href="https://warpcast.com/brehm"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="block p-2 hover:bg-base-300 rounded-lg"
+            >
+              Warpcast
+            </Link>
+          </div>
+
+          <div className="space-y-2">
+            <Link
+              href="https://github.com/austinbrehm"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="block p-2 hover:bg-base-300 rounded-lg"
+            >
+              GitHub
+            </Link>
+          </div>
+        </div>
```
What about having a mapping here?

phipsae commented on packages/nextjs/app/builders/brehm.eth/page.tsx:
Ensure your folder is named using your address, not the ENS name.
`packages/nextjs/app/builders/[your-0xAddress]/page.tsx`

phipsae commented on packages/nextjs/app/builders/brehm.eth/page.tsx:
You might want to consider making your profile mobile responsive.
<img width="605" alt="Screenshot 2025-04-15 at 08 51 28" src="https://github.com/user-attachments/assets/9e7e6afb-9bb9-47e8-ba93-10c811ba52ca" />

phipsae reviewed (changes requested):
Thanks a lot for the page! I like it with the side panel, great idea :)  

Left some comments below.

austinbrehm commented:
Ready for another review. Notable updates:

- Added mobile responsiveness
- Used a map() to avoid boilerplate code
- Made component name specific to me
- Updated page route to address (rather than ENS name)

phipsae commented:
Nice, good job, thanks! Lets merge.
